examples/ct/structures_test.go

```diff
@@ -28,49 +27,33 @@ import (
 	spb "github.com/google/trillian/crypto/sigpb"
 )
 
-var fixedTime = time.Date(2017, 9, 7, 12, 15, 23, 0, time.UTC)
-
-// Public key for Google Testtube log, taken from CT Github repository
-const ctTesttubePublicKey string = `
+var (
+	fixedTime       = time.Date(2017, 9, 7, 12, 15, 23, 0, time.UTC)
+	ctTesttubeLogID = [32]byte{176, 204, 131, 229, 165, 249, 125, 107, 175, 124, 9, 204,
+		40, 73, 4, 135, 42, 199, 232, 139, 19, 44, 99, 80, 183, 198, 253, 38, 225, 108, 108, 119}
+	// Public key for Google Testtube log, taken from CT Github repository
+	ctTesttubePublicKey = `
```

Comment:
Yes, purely for aesthetic purposes since it was the only const, and the associated vars - `ctTesttubeLogID` need to be a var because it is a slice.